src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -22,7 +25,8 @@ object Ledger extends Logger {
       log.debug(s"All txs from block ${block.header} were executed")
 
       val worldToPersist = payBlockReward(Config.Blockchain.BlockReward, block, resultingWorldStateProxy)
-      val afterExecutionBlockError = validateBlockAfterExecution(block, worldToPersist)
+      val receipts = Seq() //FIXME: This should be obtained from block execution (done in EC-139)
```

Comment:
It involves a small change so I think that any of the two PRs could be merged first. When that's the case I will update the use of the calculated receipts for the validation after the execution.